src/components/checkbox/index.tsx

```diff
@@ -159,14 +166,15 @@ class Checkbox extends Component<CheckboxProps, CheckboxState> {
     }
   };
 
-  getColor() {
-    const {color, disabled} = this.props;
-    return disabled ? DEFAULT_DISABLED_COLOR : color || DEFAULT_COLOR;
-  }
+  getColor = () => this.props.disabled ? DEFAULT_DISABLED_COLOR : this.props.color || DEFAULT_COLOR;
+
+  getBackgroundColor = () => this.props.outline ? 'transparent' : this.getColor();
+
+  getTintColor = () => this.props.outline ? this.getColor() : DEFAULT_ICON_COLOR;
```

Comment:
Hi, 
This still has issue. 
It ignore the `iconColor` prop and use the `color` prop instead. 
See my code suggestion below
```suggestion
  getTintColor = () => {
    const {outline, disabled, iconColor} = this.props;
    let tintColor = iconColor;
    if (outline) {
      tintColor = disabled ? DEFAULT_DISABLED_COLOR : tintColor || DEFAULT_COLOR;
    } else {
      tintColor = disabled ? Colors.white : tintColor || Colors.white;
    }
    return tintColor;
  }
```

Reply:
I understand now.  I changed this suggestion slightly in my commit to use more if/else instead of terniary operators.  I find it more readable now.  Hope that's ok!